Fundamentals/Setup/Server-Setup/azure-web-apps.md

```diff
@@ -61,3 +61,31 @@ It's important to know that Azure Web Apps may move your website between their '
 
 When your site is migrated to another worker, these variables will change.
 You cannot rely on these variables remaining static for the lifetime of your website.
+
+## Deploying to Linux App Service
+
+To avoid cross compatibility issues its recommeded to deploy to Linux App Service using a managed Dev Ops service such as Azure DevOps and configure a Linux build agent. 
```

Comment:
```suggestion
To avoid cross compatibility issues its recommended to deploy to Linux App Service using a managed Dev Ops service such as Azure DevOps and configure a Linux build agent. 
```
typo!

Reply:
Thanks @marcemarc